CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/ImportApiHelperTests.cs

```diff
@@ -50,5 +54,18 @@ public void AddDocumentsToWorkspaceTest(string fileType, int numberOfFiles)
 			//Cleanup
 			WorkspaceHelper.DeleteSingleWorkspaceAsync(workspaceArtifactId);
 		}
+
+		private void CleanupWorkspaceIfItExists(string workspaceName)
+		{
+			try
+			{
+				int existingWorkspaceArtifactId = WorkspaceHelper.GetFirstWorkspaceArtifactIdQueryAsync(workspaceName).Result;

```

Comment:
It's possible there could be multiple workspaces with the same name. Can we update it to delete all workspaces with the same name.